src/content/es/fundamentals/codelabs/your-first-pwapp/index.md

```diff
@@ -846,68 +852,70 @@ Además de cualquier IU que agregue para instalar su aplicación, los usuarios t
 window.addEventListener('appinstalled', logAppInstalled);
 ```
 
-Luego, necesitaremos actualizar la función `logAppInstalled` , para este código de código, solo usaremos `console.log` , pero en una aplicación de producción, probablemente desee registrar esto como un evento con su software de análisis.
+Después, necesitaremos actualizar la función `logAppInstalled`, para este laboratorio de código, solo usaremos `console.log`, pero en una aplicación de producción, probablemente desee registrar esto como un evento con su software de analíticas.
```

Comment:
`probablemente desearás registrar...` to keep the conjugation in the second person.

Reply:
Changed by `probablemente desearás registrar esto como un evento en tu software de analíticas`